website/common/script/ops/buy/buyMysterySet.js

```diff
@@ -23,10 +23,6 @@ module.exports = function buyMysterySet (user, req = {}, analytics) {
     throw new NotFound(i18n.t('mysterySetNotFound', req.language));
   }
 
-  if (typeof window !== 'undefined' && !req.noConfirm && window.confirm) { // TODO move to client
```

Comment:
Why is this removed?

Reply:
see discussion at https://github.com/HabitRPG/habitica/pull/9714/files/89f1ad2b07e0115ab494c3529518cb96ebc8d6d3..dd5870903f41d10531c4eca77c6ac4d6815d8055#diff-e92347dc0fb9e7ddffb5912653a501df